README.md

```diff
@@ -67,13 +67,13 @@ end
 
 
   kind = case year
-         when 1850..1889 then "Blues"
-         when 1890..1909 then "Ragtime"
-         when 1910..1929 then "New Orleans Jazz"
-         when 1930..1939 then "Swing"
-         when 1940..1950 then "Bebop"
-         else "Jazz"
-         end
+  when 1850..1889 then "Blues"
+  when 1890..1909 then "Ragtime"
+  when 1910..1929 then "New Orleans Jazz"
+  when 1930..1939 then "Swing"
+  when 1940..1950 then "Bebop"
+  else "Jazz"
+  end
```

Comment:
As we check against the new code only, I thought it would make sense to make sure we were up-to-date on that one. I mean, Rubocop is a living coding style anyway, right? :smiley: 
Fine with both though.